utils/cryptography/encryption.go

```diff
@@ -0,0 +1,54 @@
+package crytography
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"io"
+	"os"
+
+	"golang.org/x/crypto/nacl/secretbox"
+)
+
+// EncryptionKey for encrypting secrets
+var EncryptionKey, _ = base64.StdEncoding.DecodeString(os.Getenv("ENCRYPTION_KEY"))
+
+// Both what the encryption key length should be
+var keyLength = 32
+
+// EncryptMessage uses SecretBox to encrypt the message
+func EncryptMessage(field []byte) (encrypted []byte, nonceString [24]byte, err error) {
+	var nonce [24]byte
+
+	// The key argument should be 32 bytes long
+	if len(EncryptionKey) != keyLength {
+		return nil, nonce, errors.New("Encryption Key is not the correct key length")
+	}
+
+	var encryptionKey [32]byte
+	copy(encryptionKey[:], []byte(EncryptionKey))
+
+	if _, err := io.ReadFull(rand.Reader, nonce[:]); err != nil {
+		return nil, nonce, err
+	}
+
+	var out []byte
```

Comment:
i think you can allocate this to be size of field + secretbox.Overhead

Reply:
https://github.com/brave-intl/bat-go/pull/347/commits/bcff5e2997986b9492312cea726685a0faa130ca